cmd/healthcheck/main_test.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestHealthStatusHealthy(t *testing.T) {
+	if os.Getenv("FLAG") == "1" {
```

Comment:
The logic inside of this conditional looks pretty similar throughout the three test cases. Can we extract it into their own function?

Reply:
Thanks. Updated.